apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.cs

```diff
@@ -137,14 +167,26 @@ internal V1.Type ToProtobufType()
 
         /// <summary>
         /// Creates an array of the specified type. This can be done on any arbitrary <see cref="SpannerDbType"/>.
+        /// You may use any type that implements IEnumerable as a source for the array.  The type of each
+        /// item is determined by <paramref name="elementType"/>.
+        /// When calling <see cref="SpannerDataReader.GetFieldValue(string)"/> the default type
+        /// is <see cref="ArrayList"/>. You may, however, specify any type that implements IList which
```

Comment:
not particularly.  I did not want to force a generic on folks by default.  I can see about using List<object>